Bitwise-Operators.md

```diff
@@ -0,0 +1,27 @@
+##Bitwise Operators
+
+<< , >> , & , | , ~, and ^ are the bitwise operators which operates on one or more bit patterns or binary numerals at the level of their individual bits.
```

Comment:
~~<< , >> , & , | , ~, and ^~~ `<<` , `>>` , `&` , `|` , `~`, and `^`

~~operates~~ operate